### DISCUSSION TITLE
scrape job offers

### DISCUSSION THREAD

**Discussion by 5uru** [kevinzg/facebook-scraper#643] [Q&A]
hello do i have the possibility to scrape job offers

**Comment by neon-ninja** [comment]
Do you mean this URL? https://m.facebook.com/jobs/. The scraper does not currently support this.

**Reply by neon-ninja** [reply]
It's going away on the 22nd anyway https://www.facebook.com/business/help/982945655901961?notif_id=1643144403054354&notif_t=qp_notifs_jobs_deprecation_notif_444803306&ref=notif
